procedures: native user auth docs (#2187)
codenvy/che-docs: modules/administration-guide/partials/con_che-operator.adoc

Creates and controls the necessary {orch-name} objects to run a {prod-short} instance, such as pods, services, and persistent volumes.

`CheCluster` custom resource (CR)::

> On a cluster with the {prod-short} operator, it is possible to create a `CheCluster` custom resource (CR). The {prod-short} operator ensure full lifecycle management of the {prod-short} server components on this {prod-short} instance:

Reviewer: The {prod-short} operator ensures the full lifecycle...

Author: fixed

`CheCluster` custom resource (CR)::
On a cluster with the {prod-short} operator, it is possible to create a `CheCluster` custom resource (CR). The {prod-short} operator ensure full lifecycle management of the {prod-short} server components on this {prod-short} instance.

> The {prod-short} operator is also provides routing for {devworkspace} by configuring the `che-gateway`.

Reviewer 2: Does it mean the {prod-short} operator creates an instance of the `che-gateway` before you create a `CheCluster` CR? I thought the `che-gateway` was part of a `CheCluster` CR deployment.

If that's not the case, then please remove this line, it is confusing.

Author: I don't understand your question. che-operator deploys and configure che-gateway. I've put this sentence here, because che-operator is now part of devworkspace startup process where it provides routing for the devworkspace. It does so by configuring the che-gateway.

Reviewer 2: I believe that:

When you install Che operator, you get:

* CheCluster CRD

When you create a CheCluster CR, you get:

* Che server side services, including Che gateway and Devworkspace operator

And not: 

When you install Che operator, you get:

* CheCluster CRD
* Che gateway
* Devworkspace operator

When you create a CheCluster CR, you get:

* Che server side services

Author: ah, I think I'm getting closer to understand what you mean. Yes, you need to create `CheCluster` CR instance to get all che deployment (che-server, che-gateway, registries ... I'm not 100% sure about devworkspace-operator).

What I wanted to emphasize here is that che-operator is now not only responsible for Che deployment, but it also touches workspace startup process by configuring routing for workspaces.

> The {prod-short} operator is also provides routing for {devworkspace} by configuring the `che-gateway`.

Reviewer 2: ```suggestion
```

Reviewer 2: > che-operator is now part of devworkspace startup process where it provides routing for the devworkspace. It does so by configuring the che-gateway.

But devworkspace is installed only when you create a CheCluster CR, right?

Author: Ah, I mean workspace. It is just with devworkspace operator, you create workspace by creating Devworkspace CR so I've called it devworkspace. TBH I don't know what's the correct wording anymore. But here I meant really the user's workspaces, that he/she will create from the devfile.

> On a cluster with the {prod-short} operator, it is possible to create a `CheCluster` custom resource (CR). The {prod-short} operator ensure full lifecycle management of the {prod-short} server components on this {prod-short} instance.
> 
> The {prod-short} operator is also provides routing for {devworkspace} by configuring the `che-gateway`.

Reviewer 2: ```suggestion
On a cluster with the {prod-short} operator, it is possible to create a `CheCluster` custom resource (CR). The {prod-short} operator ensure full lifecycle management of the {prod-short} server components on this {prod-short} instance:
+
* xref:devworkspace-operator.adoc[]
* xref:gateway.adoc[]
* xref:dashboard.adoc[]
* xref:devfile-registries.adoc[]
* xref:che-server.adoc[]
* xref:postgresql.adoc[]
* xref:plug-in-registry.adoc[]
```

.Additional resources

* xref:installation-guide:configuring-the-che-installation.adoc[]
* xref:installation-guide:installing-che.adoc[]

> * xref:administration-guide:gateway.adoc[]

Reviewer 2: See comment above.

> * xref:administration-guide:gateway.adoc[]

Reviewer 2: ```suggestion
```
